From 90db3dada56a1b4239237cbcd392ce28d64659ce Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:33 -0400
Subject: [redhat] kabi: Adding symbol param_set_long (kernel/params.c)

Message-id: <4650b8caad21acfd472fbda6bb9140931db66f4a.1410510773.git.poros@redhat.com>
Patchwork-id: 96116
O-Subject: [PATCH RHEL7.1 153/233] kabi: Adding symbol param_set_long (kernel/params.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Not used directly, used by module_param internally for type long.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_long
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_long
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_long

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_long b/redhat/kabi/kabi-module/kabi_ppc64/param_set_long
new file mode 100644
index 0000000..9717f93
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_long
@@ -0,0 +1,2 @@
+#1-
+0x617643a2 param_set_long vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_long b/redhat/kabi/kabi-module/kabi_s390x/param_set_long
new file mode 100644
index 0000000..9717f93
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_long
@@ -0,0 +1,2 @@
+#1-
+0x617643a2 param_set_long vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_long b/redhat/kabi/kabi-module/kabi_x86_64/param_set_long
new file mode 100644
index 0000000..9717f93
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_long
@@ -0,0 +1,2 @@
+#1-
+0x617643a2 param_set_long vmlinux EXPORT_SYMBOL
-- 
1.7.1